src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -913,13 +913,32 @@ struct protobuf_schema_definition::impl {
           raw_extensions, std::less{}, [](const auto& extension) {
               return std::make_pair(extension.extendee(), extension.number());
           });
+
+        const auto start_section = [indent, &os](const std::string& ext) {
```

Comment:
Proto are moving to string views in general, so to avoid string creation if that happens in a later version
```suggestion
        const auto start_section = [indent, &os](std::string_view ext) {
```